Design.md

```diff
@@ -15,12 +15,29 @@ Features:
 4. Create Film Clips
     - While watching a video, user is able to clip part of a video and make a note about what happens
     - If this user is verified with a team, when another team member is watching the same video they will be able to see the note.
-    - Film clips are kept in a team-wide / user-wide database that can be searched / sorted by keywords (ie. players involved, topic discussed (ie. deep defense), teams, etc...)
-5. Allow Highlights to be Shared
+    - It is possible for a user to tag people and keywords. When a user is tagged they will be notified. Keywords can be searched for in the search field. 
+    - Global keywords (like team names, season and tournament) will be automatically added to all subsequent clips that are made in that game.
+    - After the initial clipping takes place the user is able to manipulate the start and stop timestamps if they messed up.
+5. Quality of Life Features for Film Review Page
+    - You are able to change settings when watching back a game. You can turn off comments completely. You can also filter to only show notes made by a certain author, only show notes with certain keywords or notes that you're tagged in.
+    - Clips are recorded as a duration, not a single time stamp. 
+    - If multiple clips span the same time. A queue will be formed. This will allow the user to decide if they want to watch that clip again or just move on without too much worry.
```

Comment:
We talked about this a little bit -- my gut feeling is that it's better to watch a clip one time with multiple comments to think about rather than watching a clip multiple times, each with one comment to think about.